pkg/controllers/dnsrecord/dnsrecord_controller.go

```diff
@@ -156,45 +164,28 @@ func (r *DNSRecordReconciler) deleteRecord(ctx context.Context, dnsRecord *v1alp
 // DNSRecord (dnsRecord.Status.ParentManagedZone).
 func (r *DNSRecordReconciler) publishRecord(ctx context.Context, dnsRecord *v1alpha1.DNSRecord) error {
 
-	managedZone, err := r.getDNSRecordManagedZone(ctx, dnsRecord)
+	managedZone, err := r.HostService.GetDNSRecordManagedZone(ctx, dnsRecord)
 	if err != nil {
-		return err
+		// If the Managed Zone isn't found, just continue
+		return client.IgnoreNotFound(err)
```

Comment:
why do we ignore not found here? Can there be a valid reason there is no managedzone for a dnsrecord?

Reply:
good catch , its a copy and paste error from a different function in that file calling it, ill change that

Reply:
its being used in deleteRecord function im persumming its in there in a scenario if a person deletes things out of order and deletes a managedzone the dnsrecord will still delete